docs/index.html

```diff
@@ -14,191 +14,168 @@
+                        </div>
+                        <div class="features__feature__text">
+                            Have an entire fleet? Allocate idle workers as they become available.
+                        </div>
+                        <div class="features__feature__text">
+                            Workers are designed to come and go, and can be stopped or started alongside availability and demand.
+                        </div>
+                    </div>
+                </div>
+                <div class="features__feature">
+                    <div class="features__feature__header">
+                        <div class="features__feature__title">Free for non-commercial use</div>
+                    </div>
+                    <div class="features__feature__body">
+                        <div class="features__feature__text">
+                            We love (and started from) research and open source. Any non-commercial use of HypoFuzz is completely free.
+                        </div>
+                        <a href="./docs/quickstart.html" class="button button--primary" style="width: 155px; margin-top: 1rem; margin-bottom: 1rem;">Get started <img src="./assets/arrow-right-long.svg" class="button__arrow"></a>
+                        <div class="features__feature__text">
+                            Not only that, we'd be delighted to chat or offer advice – <a href="#" class="muted-link">reach out to us</a>!
+                        </div>
+                    </div>
                 </div>
-                <div class="col-md-4 d-flex flex-column">
-                    <h2>Hypothesis</h2>
-                    <p>
-                        Hypothesis is world-leading library for property-based testing, and used by thousands of
-                        companies and Python projects who are serious about finding bugs.
-                    </p><a class="mt-auto btn btn-secondary" href="https://hypothesis.works" role="button">Learn more
-                        &raquo;</a>
+                <div class="features__feature">
+                    <div class="features__feature__header">
+                        <div class="features__feature__title">Built by the Hypothesis maintainers, with care</div>
+                    </div>
+                    <div class="features__feature__body">
+                        <div class="features__feature__text">
+                            We love testing. <a href="https://github.com/HypothesisWorks/hypothesis" class="muted-link">Hypothesis</a> is (and always will be) open-source, because we love making high-quality tooling available to everyone.
+                        </div>
+                        <div class="features__feature__text">
+                            We built HypoFuzz because we saw an opportunity to extend Python's world-leading property-based testing library, and provide even better testing tools for businesses.
+                        </div>
```

Comment:
Single paragraph for both these sentences, I think.

Reply:
I'm torn, split is easier to read, but combined does convey intent better (they're semantically grouped sentences)